test/index.js

```diff
@@ -16,7 +16,7 @@ const typedData = {
       ],
       Mail: [
           { name: 'from', type: 'Person' },
-          { name: 'to', type: 'Person' },
+          { name: 'to', type: 'Person[]' },
```

Comment:
when creating the typeHash a lookup is performed based on the type. In this case the type would be `Person[]`, so it would not match `Person` in the types dictionary.

In the test this works since `from` references `Person`, but would be interesting if it still works when we remove that reference.